utils/constants.tsx

```diff
@@ -2209,6 +2214,13 @@ export const InsightsCardTitles = {
     },
 };
 
+export enum WysiwygPluginNames {
```

Comment:
Do these need to live here or is there somewhere they could closer to the WYSIWYG editor itself? It looks like they're only used within that folder, so I'd like to see them moved there